src/components/DeploymentsModal.tsx

```diff
@@ -56,25 +56,30 @@ export const DeploymentsModal = ({ handleCloseModal, isModalOpen }: IDeployments
 
   useEffect(() => {
     // fetch deployments
-    fetchDeployments('no-cache', settings.namespace)
-      .then((output) => {
-        setDeployments(output);
-      })
-      .catch((e) => {
-        throw Error(e);
-      });
+    if (settings.namespace !== '') {
+      console.log('fetching namespace', settings.namespace)
```

Comment:
@mmelko - I think you left this here by mistake maybe?

Reply:
yep, thanks! 
It should be removed now